ivy/src/main/java/sbt/mavenint/SbtPomExtraProperties.java

```diff
@@ -12,7 +12,11 @@
     public static final String POM_INFO_KEY_PREFIX = "info.";
     public static final String POM_SCALA_VERSION = "scalaVersion";
     public static final String POM_SBT_VERSION = "sbtVersion";
+    /** @deprecated Use {@link #POM_SCALA_API_KEY} instead */
+    @Deprecated
```

Comment:
I don't know if deprecating `info.apiURL` given that there are existing poms out there is worth the change for consistency.
1. We would have to support both `info.apiURL` and `info.scalaApiURL`.
2. If someone had written a tool on using `info.apiURL`, we'll break it.

@jsuereth  Am I being too paranoid?